src/store/index.ts

```diff
@@ -16,6 +16,7 @@ export type GlobalState = {
   toasts: Toast[];
   displayedYear: number | null;
   module: ModuleJa | null;
+  tableTimeMode: Boolean;
```

Comment:
名称は変えていいけどbooleanだとわからないので変更お願いします
localstrageで使う値もここで利用する文字列で
```suggestion
  tableTimeMode: "default" | "special";
```

Reply:
大学院生用と同じようなかんじをイメージしてbooleanにしてみたのですがどうでしょうか？

<img width="477" alt="スクリーンショット 2021-04-04 16 02 44" src="https://user-images.githubusercontent.com/68944024/113501342-3e9f6f80-955f-11eb-8f41-c735d53409bf.png">

Reply:
なんか違うの見てたわ
オッケーですあと`boolean`